src/CMakeLists.txt

```diff
@@ -76,11 +76,21 @@ find_package(PythonLibs 2.7 REQUIRED)
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/treecall2.py.in"
   "${CMAKE_CURRENT_BINARY_DIR}/treecall2.py" @ONLY)
   
+file(COPY "${CMAKE_CURRENT_SOURCE_DIR}/treecall/"
+   DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/treecall/")
+
```

Comment:
Why do you copy the directory to the binary directory?  Does setuptools not allow out-of-source builds?